### PR TITLE
Carousel for homepage

### PR DESCRIPTION
**Pull request by spetruzza**
This is an attempt to replace the current banner in the homepage with a carousel (aka image slider) for the homepage. It currently contains 3 slides for covid19, scivis and vast contests.
This includes a new yaml file for configuration for the entries (_data/carousel.yml) and responsive style sheets.
It currently uses jquery and bootstrap JS from external sources, we can consider making a local copy of those in our repo.

### CODE REVIEW

File: src/styles/carousel.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* Carousel */
```
Comment (yelper) [inline src/styles/carousel.css:1]: where did the style for this come from?  is just mostly overriding the default bootstrap carousel stuff?

File: _includes/head.html [hunk 2]
```diff
@@ -7,6 +7,11 @@
     <!-- TODO: configure SEO/OG tags -->
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
+    <!-- Bootstrap 4 includes -->
```
Comment (yelper) [inline _includes/head.html:10]: ideally we'd only include these scripts on pages that need it -- can we condition this on the page calling the include?

File: src/styles/carousel.css (hunk 1 above)
Comment (spetruzza) [reply src/styles/carousel.css:1]: I did extract only the CSS parts that I needed for the carousel, without including the entire bootstrap CSS file (which would have messed up the style of the website)

File: _includes/head.html (hunk 2 above)
Comment (spetruzza) [reply _includes/head.html:11]: Yes I added it in the next commit

**Comment by yelper** [comment]
@spetruzza I pushed an update to make the whole title a flexbox layout.  Here's what it looks like: 

https://i.imgur.com/Rw7RUio.mp4

**Comment by spetruzza** [comment]
Sounds good! If you are ok we could merge this pull request

**Pull request merged**